observability/tools/readme.md

```diff
@@ -11,3 +11,20 @@ There are a number of modern tools to make systems observable. While identifying
 
 - [Loki](./loki.md)
 - [OpenTelemetry](./OpenTelemetry.md)
+
+## Service Mesh
+
+Leveraging a Service Mesh that follows the [Sidecar Pattern](https://www.oreilly.com/library/view/designing-distributed-systems/9781491983638/ch02.html#:~:text=The%20sidecar%20pattern%20is%20a,first%20is%20the%20application%20container.&text=In%20addition%20to%20the%20application,without%20the%20application%20container's%20knowledge.) quickly sets up a go-to set of metrics, and traces (although traces need to be propagated from incoming requests to outgoing requests manually).
+
+A sidecar works by intercepting all incoming and outgoing traffic to your image. It then adds trace headers to each request and emits a standard set of logs and metrics. These metrics are extremely powerful for observability, allowing every service to leverage a unified set of metrics, including:
+
+All metrics exist for both server side and client side:
+
+- Latency
+- Bytes
+- Request Rate
+- Error Rate
+
+In a microservice architecture, pinpointing the root cause of a spike in 500's can be non-trivial, but with a the added observability from a sidecar you can quickly determine which server in your service mesh resulted in the spike in errors.
```

Comment:
Guessing "server" in this line is intended to be "service," which keeps the meaning and terminology more consistent

Reply:
done!